translate.go

```diff
@@ -107,8 +107,9 @@ func SetLocale(locale string) (string, error) {
 	if matched, _ := regexp.MatchString(localeRegexp, locale); !matched {
 		return "", fmt.Errorf("Malformated locale string %s", locale)
 	}
-	locale = strings.Replace(locale, "-", "_", -1)
-	parts := strings.Split(locale, "_")
+	const sep = "-"
```

Comment:
Might as well make this a global constant.

Reply:
To me this is clearer TBH. It's the only place it's used, and if it's global the reader would have to then search for where it's used. This way is more obvious.

Reply:
I'll buy that.